css/styles.css

```diff
@@ -21014,7 +21014,7 @@ a:hover {
 
 
 .testimonial-card{
-  min-height: 358px;
+  min-height: 378px;
```

Comment:
Not sure why need to increase min-height it's not the proper solution that you are looking for.
Find a way to have consistent height across testimonials